app/controllers/v0/sessions_controller.rb

```diff
@@ -16,57 +14,73 @@ class SessionsController < ApplicationController
     # @type is set automatically by the routes in config/routes.rb
     # For more details see SAML::SettingsService and SAML::URLService
     def new
-      type = params[:signup] ? 'signup' : params[:type]
-      if REDIRECT_URLS.include?(type)
-        url = url_service.send("#{type}_url")
-
-        # If a clientId param exists, include GA clientId for cross-domain analytics
-        client_id = params[:clientId]
```

Comment:
I think this is a bad merge. it undoes the work from #2892

Reply:
yea, i definitely screwed something up... i'll have to close this and cherry pick.